packages/flutter_tools/templates/module/ios/library/Flutter.tmpl/podhelper.rb

```diff
@@ -61,11 +61,11 @@ def flutter_root(f)
 
 # Ensure that ENABLE_BITCODE is set to NO, add a #include to Generated.xcconfig, and
 # add a run script to the Build Phases.
-post_install do |installer|
+def update_configs(installer, framework_dir)
+    
     installer.pods_project.targets.each do |target|
         target.build_configurations.each do |config|
             config.build_settings['ENABLE_BITCODE'] = 'NO'
-            next if  config.base_configuration_reference == nil
```

Comment:
Why are we removing this line?

Reply:
Sorry :( I was referring an old version of flutter and found this fix. Used that code here and missed #25079 in between.

Added it back.